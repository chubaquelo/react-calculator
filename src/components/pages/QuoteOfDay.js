import React from 'react';

export default function QuoteOfDay() {
  return (
    <div className="text-center">
      <h1 className="text-3xl text-center mb-4">Quote of the Day</h1>
      <p className="text-lg">
        Without mathematics, there&apos;s nothing you can do. Everything around
        you is mathematics. Everything around you is numbers.
      </p>
      <h3 className="font-bold">Shakuntala Devi</h3>
      <img className="mt-4 mx-auto" src="./maths.jpg" alt="Maths" />
    </div>
  );
}
