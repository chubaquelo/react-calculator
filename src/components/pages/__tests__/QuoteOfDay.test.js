import React from "react";
import renderer from "react-test-renderer";
import QuoteOfDay from "../QuoteOfDay";

describe("Test QuoteOfDay Snapshot", () => {
  it("matches QuoteOfDay snapshot", () => {
    const forTest = renderer.create(<QuoteOfDay></QuoteOfDay>).toJSON();
    expect(forTest).toMatchSnapshot();
  });
});
