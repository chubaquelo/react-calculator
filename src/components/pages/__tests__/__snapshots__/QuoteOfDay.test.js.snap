// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test QuoteOfDay Snapshot matches QuoteOfDay snapshot 1`] = `
<div
  className="text-center"
>
  <h1
    className="text-3xl text-center mb-4"
  >
    Quote of the Day
  </h1>
  <p
    className="text-lg"
  >
    Without mathematics, there's nothing you can do. Everything around you is mathematics. Everything around you is numbers.
  </p>
  <h3
    className="font-bold"
  >
    Shakuntala Devi
  </h3>
  <img
    alt="Maths"
    className="mt-4 mx-auto"
    src="./maths.jpg"
  />
</div>
`;
